### ISSUE TITLE
inconsistent information about services.nginx.stateDir in 20.09 release notes

### ISSUE THREAD

**Issue by sbourdeauducq** [NixOS/nixpkgs#102211]
20.09 release notes say:

"Nginx web server now starting with additional sandbox/hardening options. **By default, write access to services.nginx.stateDir is allowed**."

But then later on say that the option has been removed:

"The Nginx log directory has been moved to /var/log/nginx, the cache directory to /var/cache/nginx. **The option services.nginx.stateDir has been removed.** "

**Comment by rnhmjoj** [comment]
pinging maintainers: @danbst  @mbbx6spp
and contributors to the stateDir deprecation: @arianvp @izorkin

**Comment by Izorkin** [comment]
Sorry, actual information:
```
   <listitem>
     <para>
       Nginx web server now starting with additional sandbox/hardening options. By default, write access
       to <literal>/var/log/nginx</literal> and <literal>/var/cache/nginx</literal> is allowed. To allow writing to other folders,
       use <literal>systemd.services.nginx.serviceConfig.ReadWritePaths</literal>
       <programlisting>
systemd.services.nginx.serviceConfig.ReadWritePaths = [ "/var/www" ];
       </programlisting>
     </para>
   </listitem>
```

**Comment by nh2** [comment]
Will be fixed as part of PR #103147.
